databuilder/databuilder/extractor/oracle_metadata_extractor.py

```diff
@@ -0,0 +1,48 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import (  # noqa: F401
+    Any, Dict, Iterator, Union,
+)
+
+from pyhocon import ConfigFactory, ConfigTree  # noqa: F401
+
+from databuilder.extractor.base_oracle_metadata_extractor import BaseOracleMetadataExtractor
+
+
+class OracleMetadataExtractor(BaseOracleMetadataExtractor):
+    """
+    Extracts Oracle table and column metadata from underlying meta store database using SQLAlchemyExtractor
+    """
+
+    def get_sql_statement(self, use_catalog_as_cluster_name: bool, where_clause_suffix: str) -> str:
+        if use_catalog_as_cluster_name:
+            cluster_source = "c.table_catalog"
+        else:
+            cluster_source = f"'{self._cluster}'"
+
+        return """
+        SELECT 
+            {cluster_source} as "cluster",
+            lower(c.OWNER) as "schema",
```

Comment:
can you unify all table & column names to either lower/upper case? I think oracle doesn't have case-sensitive column and table names so this sql could be cleaner

Reply:
Tks. Done!